### ISSUE TITLE
Label 'active' form depending on curator action

### ISSUE THREAD

**Issue by vanaukenk** [geneontology/simple-annoton-editor#97]
From 2019-08-09 call:

We would like for curators to be clear on exactly what action they are taking with any given view of form 2.0.

To help with that, we'll add a labels to the top of the form.  Here are the suggestions:

When creating a new activity in the default form:

Activity Form -> Create New Activity

When editing an existing activity:

Activity Form -> Edit Existing Activity

When creating a new BP only:

Activity Form -> Create New BP Annotation

When creating a new CC only:

Activity Form -> Create new CC Annotation

I'd love to have a different name for what we are now calling the 'Default Activity' form.  We've moved away from using the 'Annoton' terminology and now use 'Activity Unit' in the Google doc derived from the ShEx.  Any thoughts?

Also, the BP only form also creates a root MF annotation, but I think it's okay for now to just say 'Create New BP Annotation'.

**Comment by tmushayahama** [comment]
@vanaukenk I was thinking we can now start to put new issues in this repo https://github.com/geneontology/noctua-form

Activity Unit sounds good. 

**Just for a fun ramble** 
Create and New in most apps, either one of them is used, but not both :) . 
UI discussion here https://ux.stackexchange.com/questions/57535/new-vs-create 

Verbs are always good, so there's another nice one  'Add' It actually cover both "Create" and "new" . In overall Noctua terminology, it might be create a new model and Add new Activity Unit in a model
Add new BP Annotation in it etc

**Comment by vanaukenk** [comment]
Yes, @tmushayahama - I will put new tickets in noctua-form now.  Sorry about that!

Let's discuss exact terminology on tomorrow's workbenches call :-)

**Comment by pgaudet** [comment]
Out of date.
